### PR TITLE
Security fixes 

### PR DESCRIPTION
**Pull request by Anusha913082**

### CODE REVIEW

File: admin/class-wpadcenter-admin.php [hunk 1]
```diff
@@ -3794,7 +3794,7 @@ public function wpadcenter_admin_review_notice() {
 	 */
 	public function wpadcenter_review_already_done() {
 		$dnd   = '';
-		$nonce = isset( $_POST['wpadcenter_review_nonce'] ) ? sanitize_text_field( wp_unslash( $_POST['wpadcenter_review_nonce'] ) ) : '';
+		$nonce = isset( $_GET['wpadcenter_review_nonce'] ) ? sanitize_text_field( wp_unslash( $_GET['wpadcenter_review_nonce'] ) ) : '';
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php]: Nonce verification is required before accessing the value of GET variables.

**Review by dhananjaymayane — COMMENTED** [review]
Resolve the comments

**Review by sanketbhbwsl — APPROVED** [review]

File: admin/class-wpadcenter-admin.php [hunk 2]
```diff
@@ -3758,19 +3758,23 @@ public function wpadcenter_admin_review_notice() {
...
+					echo '<style>.wpadcenter-review-notice.updated{padding-bottom:1%;display:flex;flex-direction:column}.wpadcenter-review-btns{background-color:#2271b1;padding:10px;max-width:150px;text-align:center;border-radius:2px;color:white;}.wpadcenter-review-already-done-btn{margin-left:10px}.wpadcenter-review-btns-container{display:flex;flex-direction:row}.wpadcenter-review-btns:hover{background-color:#135e96}.wpadcenter-review-btns>a{color:#fff;text-decoration:none}.wpadcenter-review-already-done-btn>a>i,.wpadcenter-review-rate-us-btn>a>i{margin-left:5px;}.wpadcenter-review-notice-text-container{display:flex;flex-direction:row;justify-content:space-between;}.wpadcenter-review-dismiss-btn{display:flex;margin-top:10px;text-decoration:none}.wpadcenter-review-dismiss-btn>i{font-size:15px}.wpadcenter-review-dismiss-btn>.dashicons-dismiss:before{vertical-align:middle}@media (max-width:768px){.wpadcenter-review-notice.updated{padding-bottom:2%!important}}</style>';
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php]: Can we add the inline CSS to the file and then enqueue the CSS file.

**Review by dhananjaymayane — COMMENTED** [review]
Please resolve the comments

**Review by dhananjaymayane — APPROVED** [review]

**Pull request merged**